packages/ui/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@harnessio/ui",
   "description": "Harness Canary UI component library",
-  "version": "0.0.4",
+  "version": "0.0.5",
```

Comment:
let's not bump version in the same PR as code changes.. 
we can combine multiple fixes into one new release

Reply:
Yes makes sense, reverted.